apiserver/facades/controller/crossmodelrelations/crossmodelrelations.go

```diff
@@ -44,6 +44,11 @@ type CrossModelRelationsAPI struct {
 	offerStatusWatcher    offerStatusWatcherFunc
 }
 
+// CrossModelRelationsAPIv1 has WatchRelationUnits rather than WatchRelationChanges.
+type CrossModelRelationsAPIv1 struct {
```

Comment:
V1

(I think we use capital V elsewhere)

Reply:
I mean, it looks like it's a mixture - spaces, bundles, subnets, actions, controller, storage, storageprovisioner all use lowercase v...
Uppercase is more common. It's strictly worse in terms of readability though! :(

Reply:
I'll do it if you insist but I'll probably mutter about it under my breath while I do.
Admittedly this is less effective in a remote-working situation.

Reply:
I recognise now this is a weird hill to die on. Changed!